### DISCUSSION TITLE
Is it possible to use deck.gl with Here maps?

### DISCUSSION THREAD

**Discussion by kwconrad** [visgl/deck.gl#8980] [Q&A] [answered]
I want to integrate Deck.gl with Here maps but I'm not sure if there is some fundamental incompatibility. Is there anything I should be aware of, or are they both just compatible? (I'm still new to WebGL in general, so maybe I am misunderstanding something)

Thanks!

**Comment by donmccurdy** [comment, marked as answer]
The HERE documentation mentions an [deck.gl “integration”](https://www.here.com/docs/bundle/data-api-developer-guide/page/rest/layers.html), but I'm not sure what that involves or how well-supported it is. The discussion at https://github.com/visgl/deck.gl/issues/4147 may be helpful as a starting point. In general there is some work involved in maintaining basemap integrations for providers like Mapbox, Maplibre, and Google Maps. I would assume that a HERE integration is possible, but a bit of work if it needs to be done from scratch.

**Reply by Pessimistress** [reply]
There is a [get-started example for harp.gl](https://github.com/visgl/deck.gl/tree/master/examples/get-started/pure-js/harp.gl) which was HERE map's JS solution at one point. Looks like their SDK has evolved but harp is still documented as an alternative renderer. Maybe the camera parameters are still the same.

**Reply by kwconrad** [reply]
Thank you @donmccurdy , thank you @Pessimistress !
